qiskit_nature/second_q/algorithms/excited_states_solvers/qeom.py

```diff
@@ -74,6 +76,82 @@ class EvaluationRule(Enum):
     DIAG = "diag"
 
 
+def _commutator(op_a: SparsePauliOp, op_b: SparsePauliOp) -> SparsePauliOp:
+    r"""Compute commutator of `op_a` and `op_b`.
+
+    .. math::
+
+        AB - BA.
+
+    Args:
+        op_a: Operator A.
+        op_b: Operator B.
+
+    Returns:
+        The computed commutator.
+    """
+    return (op_a @ op_b - op_b @ op_a).simplify()
```

Comment:
Is it ok that simplify method does not take an argument, for example `atol=0`?

Reply:
Ah yes indeed we should force 0 instead of the default value

Reply:
Thanks, @ikkoham! I did not noticed this before and also just opened a PR to fix this in the routines in Nature: https://github.com/Qiskit/qiskit-nature/pull/1045